lib/src/_shared/DateTextField.jsx

```diff
@@ -42,6 +44,8 @@ export default class DateTextField extends PureComponent {
     mask: undefined,
     keyboardIcon: 'event',
     invalidDateMessage: 'Invalid Date Format',
+    clearable: true,
```

Comment:
Do we need few default value definitions for `clearable`? It's already defined in `ModalWrapper`

Reply:
I think it can be removed. I just tried to handle it the same way as it is for other probs like keyboard. If DateTextField is just used inside this package it is not neccessary. In the current way it could be used in other compositions.

Reply:
You are right, let's leave default values for those props